File: solutions/1018_largest-perimeter-triangle/solution.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int i;
        int a,b,c;
        for(int i=nums.size()-1;i>=2;i--){
            a = nums[i-2];
            b = nums[i-1];
            c = nums[i];
            if ((a+b)>c && (b+c)>a && (a+c)>b ){
                return a+b+c;
            }
        }
        return 0;
    }
};

File: solutions/1018_largest-perimeter-triangle/solution.py
class Solution:
    def largestPerimeter(self, nums: List[int]) -> int:
        nums = sorted(nums,reverse =True)
        for i in range(len(nums)-2):
            a,b,c = nums[i:i+3]
            if(a+b > c and b+c > a and a+c > b):
                return a+b+c
        return 0

